sbx/crossq/batch_renorm.py

```diff
@@ -0,0 +1,188 @@
+from typing import Any, Callable, Optional, Sequence, Tuple, Union
+
+import jax
+import jax.numpy as jnp
+from flax.linen.module import Module, compact, merge_param  # pylint: disable=g-multiple-import
+from flax.linen.normalization import _canonicalize_axes, _compute_stats, _normalize
+from jax.nn import initializers
+
+PRNGKey = Any
+Array = Any
+Shape = Tuple[int, ...]
+Dtype = Any  # this could be a real type?
+Axes = Union[int, Sequence[int]]
+
+
+class BatchRenorm(Module):
```

Comment:
maybe add a quick explanation of why we use it instead of batch norm.
I would also put that in `common/jax_layers.py`